lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -601,6 +608,7 @@ void SessionStateEnforcer::prepare_response_to_access(
   rsp_cmn->mutable_sid()->CopyFrom(config.common_context.sid());  // imsi
   rsp_cmn->set_sm_session_state(config.common_context.sm_session_state());
   rsp_cmn->set_sm_session_version(config.common_context.sm_session_version());
+  MLOG(MINFO) << "Send message to AMF gRPC client handler :: ";

```

Comment:
What is '::' ?

Reply:
Sorry, I forgot to remove logging statement. Removed it.